ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -1269,11 +1273,7 @@ objects:
           ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-bootstrap.yaml"
           ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-network.yaml"
           ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-security-groups.yaml"
-
-          # down-containers.yaml was introduced in 4.6
-          if [ -f "${ASSETS_DIR}/down-containers.yaml" ]; then
-                ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-containers.yaml" --list-tasks
```

Comment:
haha, nice. We never executed the down-containers playbook.

Reply:
I think it was executed before my changes! Let me fix the commit history, so that it's clearer